cmd/convox/scale.go

```diff
@@ -35,6 +39,7 @@ func cmdScale(c *cli.Context) error {
 	// initialize to invalid values that indicate no change
 	count := -2 // -1 is valid, indicates removing the process and ELB
 	memory := -1
+	cpu := -1
```

Comment:
This makes total sense in this patch, but adding some more context while it's fresh... Recently we got bit by the client posting these magic values to indicate no change. A better design would be that the client doesn't post the field at all if it wasn't specified.